### ISSUE TITLE
Action: Add "Save Changes" button to top white box

### ISSUE THREAD

**Issue by cecilia-donnelly** [OpenTechStrategies/swoptact#108]
The "save changes" button will be positioned with the action-wide information, rather than with the attendees on the bottom half of the page.  Both "Save Changes" and "Done" buttons will save all the information on the page, but the "Save Changes" button will submit all information, page-wide, and stay on the page, while the Done button will likewise save all the information but leave the page (exactly as it does now).

**Comment by cecilia-donnelly** [comment]
I added the new button in 5e2dbaf and it saves Action information correctly, but it doesn't yet save information from the sign-in sheet template.  I think that will require a change to inline_ajax.js, so I'm waiting for @cwebber to finish refactoring that file in #94 before I continue this issue.  It should be simple to add sign-in sheet saving to this button (before form submission) thanks to the work I did in #47.
